src/components/TransactionFlow/displays/TransactionDetailsDisplay.tsx

```diff
@@ -61,6 +69,26 @@ function TransactionDetailsDisplay({
     ? weiToFloat(bigNumberify(userAssetToSend.balance), asset.decimal).toFixed(6)
     : translateRaw('UNKNOWN_BALANCE');
 
+  const gasUsedPercentage = (() => {
+    if (gasUsed) {
+      const gasLimitBN = bigify(gasLimit);
+      const gasUsedBN = bigify(gasUsed.toString());
+      return gasUsedBN.div(gasLimitBN).multipliedBy(bigify(100));
+    }
+    return undefined;
+  })();
```

Comment:
utility pure function plz.

We aim to separate concerns between the value stored and the value displayed.
Here there are two steps. The first that caluculates the percentage. The second that formats the percentage to a display value.